client/server/server.go

```diff
@@ -45,13 +46,15 @@ type oauthAuthFlow struct {
 }
 
 // New server instance constructor.
-func New(ctx context.Context, managementURL, adminURL, configPath, logFile string) *Server {
+func New(ctx context.Context, managementURL, adminURL, configPath, logFile, wgInterfaceName string, wgPort int) *Server {
```

Comment:
We are avoiding adding config flags to the daemon service New function because it requires a reinstallation to update them.

Instead, let's also handle the update the internal.ReadConfig function. This way we can use `netbird up --wg-iface wt0 --wg-port 54321` and updating that will require just netbird down and up again.